jonprl-mode.el

```diff
@@ -48,6 +48,28 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   (":" . ?∶)
+   ("<" . ?⟨)
+   ("<>" . ?⋄)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   (">" . ?⟩)
+   ("[" . ?⸤)
+   ("]" . ?⸥)
```

Comment:
This one doesn't quite work out, since in JonPRL's syntax, we have `].`, which pretty-symbols-mode just renders as `].` instead of `⸥.`.